src/ui/FieldTable/FieldValue.ts

```diff
@@ -255,6 +275,12 @@ export class FieldValue extends Component {
     });
   }
 
+  private prependTextCaptionToDom(): void {
+    let elem = this.getValueContainer();
+    elem.insertBefore(this.renderTextCaption(), elem.childNodes[0]);
+    elem.style.display = 'inline-block';
```

Comment:
I would add a classname that describe the state of the field value, instead of the way it should be displayed. So something like this :  

    .CoveoFieldValue {
       .coveo-with-label {
            display: inline-block;
       }
    }